ietf/templates/meeting/add_session_notes_and_recordings.html

```diff
@@ -0,0 +1,70 @@
+{% extends "base.html" %}
+{# Copyright The IETF Trust 2015, All Rights Reserved #}
+{% load origin static django_bootstrap5 %}
+{% block title %}Add I-Ds to {{ session.meeting }} : {{ session.group.acronym }}{% endblock %}
+{% block pagehead %}{{ form.media.css }}{% endblock %}
+{% block content %}
+    {% origin %}
+    <h1>
+        Add Notes and Recordings to  {{ session.meeting }}
+        {% if session_number %}: Session {{ session_number }}{% endif %}
+        <br>
+        <small class="text-body-secondary">{{ session.group.acronym }}
+            {% if session.name %}: {{ session.name }}{% endif %}
+        </small>
+    </h1>
+    {% comment %} TODO: put the session name here or calculate the number at the meeting {% endcomment %}
+    {% if session.is_material_submission_cutoff %}
+        <div class="alert alert-warning my-3">
+            The deadline for submission corrections has passed. This may affect published proceedings.
+        </div>
+    {% endif %}
+    <div class="alert alert-info my-3">
+        This form will link additional Notes and Recordings to this session with a revision of "Current at time of presentation". For more fine grained control of versions, or to remove Notes and Recordings from a session, adjust the sessions associated with an Internet-Draft from the Internet-Draft's main page.
```

Comment:
Sorry - missed commenting on this earlier: it's copied from the workflow that lets chairs associate drafts with sessions, and the lifecycle of such things is not the same. With drafts, chairs sometimes want to associate an older version with a session. Here, older recordings aren't really something to point to. It's a good indication that recording as a type of document may have been a mis-model. But in any case, please just remove this paragraph.